src/test/java/apprenticeship/GameTest.java

```diff
@@ -0,0 +1,4 @@
+package apprenticeship;
+
+public class GameTest {
+}
```

Comment:
It should right? I'm just having trouble writing it. Still thinking about it....